doc/qbk/1.0.overview.qbk

```diff
@@ -95,12 +111,12 @@ these per-commit analyses:
 [/-----------------------------------------------------------------------------]
 
 [section:security_review Security Review (Bishop Fox)]
-TBA
+To be announced
 [endsect]
 
 [/-----------------------------------------------------------------------------]
 
-[h1 Credits]
+[h1 Acknowledgments]
```

Comment:
I changed this one because Credits might imply the authors while Acknowledgments imply people and institutions who helped a lot. If this gets into Boost, it might be worth mentioning the alliance in general here.